package.json

```diff
@@ -34,5 +34,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@types/styled-components": "^4.4.0"
```

Comment:
Please also re-install the following dependencies to be under `devDependencies`:

```
"@types/jest": "24.0.23",
"@types/node": "12.12.14",
"@types/react": "16.9.13",
"@types/react-dom": "16.9.4",
"typescript": "3.7.2"
```

You can do that by running this command:

```bash
$ npm install <PACKAGE_NAME> -D
```

> `-D` makes the dependency as a dev dependency. This means that you need this dependency to _develop_ the website, but don't need it for the website to run in the browser.